### DISCUSSION TITLE
Projeto ainda em aberto?

### DISCUSSION THREAD

**Discussion by samuelbozio** [grcreutzberg/dice-api#3] [General]
Sou novo no github e nem sei se essa aba serve pra eu tirar dúvidas mas, sou fã de RPG e tenho um conhecimento intermediário em Java. Se quiser mais alguém para contribuir, estou disponível. 

Meu discord: SamuelBozio#3092

**Comment by grcreutzberg** [comment]
> Sou novo no github e nem sei se essa aba serve pra eu tirar dúvidas mas, sou fã de RPG e tenho um conhecimento intermediário em Java. Se quiser mais alguém para contribuir, estou disponível.
> 
> Meu discord: SamuelBozio#3092

Ola @samuelbozio, seja bem vindo!
Também sou um entusiasta do RPG, esse projeto esta aberto para contibuições sim, fique a vontade para clonar o projeto e mandar suas sugestões de melhorias/correções.

PS: Vou transferir a Issue para [Discussions](https://github.com/grcreutzberg/dice-api/discussions).
